frontend/app-development/layout/PageHeader/SubHeader/SettingsModalButton/SettingsModal/components/Tabs/Maskinporten/ScopeListContainer/SaveStatus/SaveStatus.test.tsx

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import { SaveStatus, type SaveStatusProps } from './SaveStatus';
+import { textMock } from '@studio/testing/mocks/i18nMock';
+
+const defaultProps: SaveStatusProps = {
+  isPending: true,
```

Comment:
I suggest that we have all negative values as default.
```suggestion
  isPending: false,
```

Reply:
See comment below